requirements.txt

```diff
@@ -1,6 +1,6 @@
 Django~=2.2.6
 djangorestframework~=3.9.2
-psycopg2==2.8.3
+psycopg2-binary==2.8.4
```

Comment:
Per the pip package:

> The binary package is a practical choice for development and testing but in production it is advised to use the package built from sources.

So this isn't the best long term choice, but it's required for getting Django to run within Docker.

Reply:
Does GitHub Actions run the tests in a Docker environment? Or are you just preparing for the future?

Reply:
So the Backend Testing step is basically required to do in Docker, since we need 2 containers, one for the Postgres DB, and one for the Django server. You may be able to do it just using GitHub Actions, but doing the set up in Docker greatly reduces how long it takes to get it up and running, since if it works in a local Docker container on our PC's, then it will most likely work in the GitHub Actions CI environment(which is one of the major benefits of containerization).

Reply:
Likewise, the Backend Linting step uses [this action](https://github.com/cclauss/GitHub-Action-for-pylint), which also runs Docker.